CanalTP/Chaos
# Adding requirments on impact & disruption post data format

# Description

This PR adds more validation on data format of impact and disruption  APIs

## Issue (optional)

Issue link: BOT-536

## Pull Request type (optional)

This is a bug fix

## How to test

Here are the following steps to test this pull request:

- It should be impossible to create an disruption without impact
- It should be impossible to create an impact without pt object
